CMakeLists.txt

```diff
@@ -18,6 +18,8 @@ endif()
 set(CLR_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(VM_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src/vm)
 set(GENERATED_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/src/inc)
+set(VERSION_FILE_PATH "${CMAKE_BINARY_DIR}/../version.c")
```

Comment:
I think we should generate the version file to the CMAKE_BINARY_DIR directly, not one level below. All generated files for a configuration (release, debug, ...) should be contained to the CMAKE_BINARY_DIR.